msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -14,7 +14,7 @@ The resource-specific consent (RSC) permissions model, originally developed for
 
 > [!NOTE]
 >
-> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md).
+> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md) and will only be enabled after a new installation into a chat.
```

Comment:
Let's add a reference code for apps that want the RSC permission for Graph, but don't want to react to all messages. In the code snippet below show a sample of how to implement the previous behavior by checking for a bot mention and ignoring the message otherwise.

Reply:
That's a good idea, I added a code snippet.